eXo/Sources/Controllers/HomePage/HomePageViewController.swift

```diff
@@ -158,4 +177,49 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
         }
         return nil
     }
+    
+    /*
+    Display the Onboarding View Controller if:
+    - The view has never been shown
+    - After use has logged in
+    */
+    func showOnBoardingIfNeed () {
+        if (NSUserDefaults.standardUserDefaults().objectForKey(Config.onboardingDidShow) == nil){
+            let welcomeVC:WelcomeViewController = self.storyboard?.instantiateViewControllerWithIdentifier("WelcomeViewController") as! WelcomeViewController
+            welcomeVC.modalPresentationStyle = UIModalPresentationStyle.FullScreen
+            welcomeVC.modalTransitionStyle = UIModalTransitionStyle.CrossDissolve
+            self.presentViewController(welcomeVC, animated: true, completion: {})
+            NSUserDefaults.standardUserDefaults().setObject(NSNumber(int: 0), forKey: Config.onboardingDidShow)
```

Comment:
Can you store a boolean instead of an int?
If not, use the value 1 instead of 0.